docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -0,0 +1,380 @@
+# Installing With SR-IOV Worker Nodes
+
+## Table of Contents
+
+- [Pre-Install Requirements](#pre-install-requirements)
+- [Preparing your OpenShift Cluster to use SR-IOV Networks](#preparing-your-openshift-cluster-to-use-sr-iov-networks)
+- [Creating SR-IOV Networks for Worker Nodes](#creating-sr-iov-networks-for-worker-nodes)
+- [Creating SR-IOV Worker Nodes in IPI](#creating-sr-iov-worker-nodes-in-ipi)
+- [Creating SR-IOV Worker Nodes in UPI](#creating-sr-iov-worker-nodes-in-upi)
+
+## Pre-Install Requirements
+
+Using Single Root I/O Virtualization (SR-IOV) networking in OpenShift can be beneficial for applications
+that require high bandwidth and low latency. If you are planning an OpenStack deployment with SR-IOV NICs enabled, refer to [the OSP 16.1 installation documentation](osp-sriov-install). Before installing, make sure that the network interface cards your OpenStack nodes are using [are supported](supported-nics) for use with SR-IOV
+in OpenShift, and that your tenant has access to them. Your OpenStack cluster will also need the following requirements for each instance with an attached SR-IOV NIC:
+
+- An instance from the RHOSP quota
+- One Port attached to the machines subnet
+- One port for each SR-IOV NIC
+- A flavor with at least 16 GB memory, 4 vCPUs, and 25 GB storage space
+
+Additionally, for clusters that use single-root input/output virtualization (SR-IOV), RHOSP compute nodes require a flavor that supports [huge pages](huge-pages).
```

Comment:
@atyronesmith do all instances that have SR-IOV NICs attached need huge pages support in their flavors? If so, I will move this up to the bulleted pre-requisite list because its easier to understand that way.

Reply:
I would move it up.  The majority of use cases will require hugepages.